src/cascadia/TerminalApp/dll/TerminalApp.vcxproj

```diff
@@ -28,15 +28,15 @@
   <ItemGroup>
     <!-- Only put headers for winrt types in here. Don't put other header files
     in here - put them in TerminalAppLib.vcxproj instead! -->
-    <ClInclude Include="AppLogic.h" />
+    <ClInclude Include="../AppLogic.h" />
     <ClInclude Include="pch.h" />
-    <ClInclude Include="TerminalPage.h" />
-    <ClInclude Include="MinMaxCloseControl.h" />
-    <ClInclude Include="AppKeyBindings.h" />
-    <ClInclude Include="TitlebarControl.h" />
-    <ClInclude Include="TabRowControl.h" />
-    <ClInclude Include="App.h" />
-    <ClInclude Include="Tab.h" />
+    <ClInclude Include="../TerminalPage.h" />
+    <ClInclude Include="../MinMaxCloseControl.h" />
+    <ClInclude Include="../AppKeyBindings.h" />
+    <ClInclude Include="../TitlebarControl.h" />
+    <ClInclude Include="../TabRowControl.h" />
+    <ClInclude Include="../App.h" />
+    <ClInclude Include="../Tab.h" />

```

Comment:
does the dll project even need the ClIncludes? Can it live w/o them?

Reply:
probably?